llvm/utils/lit/lit/main.py

```diff
@@ -351,4 +353,4 @@ def print_summary(tests_by_code, quiet, elapsed):
     for (label, count) in groups:
         label = label.ljust(max_label_len)
         count = str(count).rjust(max_count_len)
-        print("  %s: %s" % (label, count))
+        print("  %s: %s (%.2f%%)" % (label, count, float(count) / total_tests * 100))
```

Comment:
Wishlist: this can probably also be converted to a f-string or if not to use .format() while you are at it. Since this pre-existing I don't mind if you leave it untouched.

Reply:
Thanks. Will take it in a separate patch.